### ISSUE TITLE
[BUG] Visual boundary of the Form -- Select Dropdowns lacks 3 to 1 color contrast

### ISSUE THREAD

**Issue by mndonx** [emulsify-ds/compound#74]
**Describe the bug**

**To Reproduce**
1. Using Chrome, Open https://emulsify-ds.github.io/compound/?path=/story/atoms-forms--select-dropdowns
2. Now measure the color contrast for the boundary of the dropdown against the background white color
3. Currently, the visual boundary (light grey border) of the dropdown lacks 3 to 1 color contrast. Label of the field will be sufficient to indicate the hit area of the field providing border contrast will help users with low vision and colorblindness

Foreground: #F2F2F2
Background: #FFFFFF
The contrast ratio is: 1.1:1

**Expected behavior**
Non text elements like boundaries, focus indicators, icons must have at least 3 to 1 color contrast against the background

People who have low vision or are colorblind may have difficulty perceiving the full hit area of the element

Ensure that boundaries have at least 3 to 1 color contrast against the background. If the boundary  color is changed to #949494. New contrast ratio: 3: 1

Refer:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast
https://accessible-colors.com/

**Screenshots**
![Visual boundary of the combobox lacks 3 to 1 color contrast](https://user-images.githubusercontent.com/2343957/208309382-bc18e6e4-d299-4a73-9f10-952e6da6339a.png)

**Desktop (please complete the following information):**
 - All

**Smartphone (please complete the following information):**
 - All
